client/routes.js

```diff
@@ -46,6 +46,7 @@ class Routes extends Component {
         {isLoggedIn && (
           <Switch>
             {/* Routes placed here are only available after logging in */}
+            <Route path="/home" component={UserHome} />
```

Comment:
Here, when the user logs in there is no UserHome.
Need to change this path to show the component {EntrancePage}